test/optimist/parser_educate_test.rb

```diff
@@ -158,6 +158,17 @@ def test_help_has_grammatical_default_text
     assert help[1] =~ /Default/
     assert help[2] =~ /default/
   end
+
+  def test_help_has_grammatical_permitted_text
+    parser.opt :arg1, 'description with period.', :type => :strings, :permitted => %w(foo bar)
```

Comment:
Does this also work with other types that are not strings?  At a minimum I was thinking :string type, but even something like an integer would be useful to constrain.  If we can have other types constrained, I think we should also have some specs for those.

Reply:
If I had known you had an OptimistXL I would have just used that instead! For this and the other PRs too.